src/controllers/observation-controller.ts

```diff
@@ -60,26 +51,46 @@ export default class ObservationController extends AbstractController {
   };
 
   private addObservation = async (req: Request, res: Response, next: NextFunction) => {
-    const newObservation = req.body;
-    if (!req.user) {
-      return res.status(401).send();
+    const rawObservation = req.body;
+    if (!req.user) res.status(401).send();
+
+    try {
+      const newObservation = await Observation.create({ ...rawObservation, finder: req.user.id });
```

Comment:
There's a question about that... It allows you to create an observation only for the user himself, or for another user in any way.

Reply:
As far as existing implementation goes, adding new observation is allowed for user himself.
However, further clarifications required.